File: app/src/main/java/com/ad/whatsappclone/ChatInfoActivity.java
package com.ad.whatsappclone;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.ad.whatsappclone.databinding.ActivityChatInfoBinding;

public class ChatInfoActivity extends AppCompatActivity {
    ActivityChatInfoBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityChatInfoBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/ad/whatsappclone/ui/chat_details/ChatInfoActivity.kt
package com.ad.whatsappclone.ui.chat_details

import android.os.Bundle
import com.ad.whatsappclone.databinding.ActivityChatInfoBinding
import com.ad.whatsappclone.ui.BaseActivity

class ChatInfoActivity : BaseActivity() {
    private lateinit var binding: ActivityChatInfoBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityChatInfoBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}